lib/command.js

```diff
@@ -1837,7 +1837,9 @@ Expecting one of '${allowedValues.join("', '")}'`);
    */
   description(str, argsDescription) {
     if (str === undefined && argsDescription === undefined) return this._description;
```

Comment:
Alternative solution that would make the change invisible to the outside world:

```suggestion
    if (str === undefined) return this._description;
```

Reply:
I actually like this alternative solution better, and `argsDescription` is deprecated. But it would technically be a breaking change. I think simpler just to leave it alone (and one day get rid of `argsDescription`).